web/packages/teleport/src/Discover/Server/EnrollEc2Instance/EnrollEc2Instance.test.tsx

```diff
@@ -104,7 +104,8 @@ describe('test EnrollEc2Instance.tsx', () => {
     expect(ctx.nodeService.fetchNodes).not.toHaveBeenCalled();
   });
 
-  test('an instance that is already enrolled should be disabled', async () => {
+  // TODO: Fix flaky network error failure in this test
+  test.skip('an instance that is already enrolled should be disabled', async () => {
```

Comment:
I didn't notice that this test has a similar problem, I guess it's for the same reason as `EnrollRdsDatabase` above.